runtime/exporter-otlp/src/main/java/io/aklivity/zilla/runtime/exporter/otlp/internal/OtlpExporterContext.java

```diff
@@ -45,7 +47,9 @@ public ExporterHandler attach(
         Collector collector,
         LongFunction<KindConfig> resolveKind)
     {
-        return new OltpExporterHandler(config, context, exporter, collector, resolveKind, attributes);
+        OtlpExporterConfig otlpExporter = new OtlpExporterConfig(config, context, exporter);
+        return new OltpExporterHandler(config, context, otlpExporter, (OtlpOptionsConfig) exporter.options,
```

Comment:
No need to pass options separately from `otlpExporter` as `otlpExporter.options` is already correct type and reachable from handler.